ArduCopter/system.cpp

```diff
@@ -262,6 +262,9 @@ void Copter::init_ardupilot()
     // ready to fly
     serial_manager.set_blocking_writes_all(false);
 
+    //Setting the mode to default initial_mode
+    set_mode((control_mode_t)g.initial_mode.get(), MODE_REASON_UNKNOWN);
```

Comment:
This reason isn't really great.

Reply:
As this is a part of the larger initialisation process of the copter code will MODE_REASON_INITIALISING be a better alternative for this. Or should we add another enum type for Mode reasons such as MODE_REASON_STABILISED or MODE_REASON_INITIALISED